src/index.js

```diff
@@ -69,6 +69,10 @@ class GifPlayerContainer extends React.Component {
 
   componentDidUpdate (prevProps, prevState) {
     this.updateImages(prevState);
+    const { onTogglePlay=()=>{} } = this.props;
```

Comment:
I guess it's more slick/compact this way but I strongly prefer checking if `typeof onTogglePlay === 'function'` to creating a new arrow function unnecessarily.

Reply:
I did ask which you preferred because I felt you'd like to check explicitly–though the other way is, as you mention, more "slick" ;)

Reply:
I do have a soft spot in my heart for those slick answers, but I generally find the other way easier to read and a bit more performant. Thanks! 🙂

Reply:
I do have a soft spot in my heart for those slick answers, but I generally find the other way easier to read and a bit more performant. Thanks! 🙂